src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -1,55 +1,56 @@
 package com.kodilla.ecommercee.controller;
 
-import com.kodilla.ecommercee.domain.CartDto;
-import com.kodilla.ecommercee.domain.OrderDto;
 import com.kodilla.ecommercee.domain.ProductDto;
-import org.springframework.http.MediaType;
+import com.kodilla.ecommercee.mapper.CartMapper;
+import com.kodilla.ecommercee.service.CartDbService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
 
-import java.math.BigDecimal;
-import java.util.ArrayList;
 import java.util.List;
 
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
 @RestController
 @RequestMapping("/v1/carts")
 public class CartController {
-    @PostMapping
-    public void create() {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CartController.class);
+    private CartMapper cartMapper;
+    private CartDbService cartDbService;
+
+    @Autowired
+    public CartController(CartMapper cartMapper, CartDbService cartDbService) {
+        this.cartMapper = cartMapper;
+        this.cartDbService = cartDbService;
     }
 
-    @GetMapping
-    public List<ProductDto> get() {
+    @PostMapping(value="createCart",consumes = APPLICATION_JSON_VALUE)
```

Comment:
```suggestion
    @PostMapping
```
 PostMapping mówi nam, że mamy utworzyć koszyk, więc nie potrzebne jest value,
    consumes też nie jest nam potrzebne, ponieważ nie przekazujemy nic w ciele posta.

Reply:
Ok, poprawione.